Integrations/IntSight/IntSight.py

```diff
@@ -328,45 +330,48 @@ def get_alert_activity():
     """
     alert_id = demisto.getArg('alert-id')
     response = http_request('GET', 'public/v1/data/alerts/activity-log/' + alert_id, json_response=True)
-
-    human_readable_arr = []
-    alert = {'ID': alert_id, 'Activities': []}
-
-    for activity in response:
-        alert['Activities'].append({
-            'ID': demisto.get(activity, '_id'),
-            'Type': demisto.get(activity, 'Type'),
-            'Initiator': demisto.get(activity, 'Initiator'),
-            'CreatedDate': demisto.get(activity, 'CreatedDate'),
-            'UpdateDate': demisto.get(activity, 'UpdateDate'),
-            'RemediationBlocklistUpdate': demisto.get(activity, 'AdditionalInformation.RemediationBlocklistUpdate'),
-            'AskTheAnalyst': {'Replies': demisto.get(activity, 'AdditionalInformation.AskTheAnalyst.Replies')},
-            'Mail': {'Replies': demisto.get(activity, 'AdditionalInformation.Mail.Replies')},
-            'ReadBy': demisto.get(activity, 'ReadBy')
-        })
-        human_readable_arr.append({
-            'ID': demisto.get(activity, '_id'),
-            'Type': demisto.get(activity, 'Type'),
-            'Initiator': demisto.get(activity, 'Initiator'),
-            'CreatedDate': demisto.get(activity, 'CreatedDate'),
-            'UpdateDate': demisto.get(activity, 'UpdateDate'),
-            'RemediationBlocklistUpdate': extract_remediation(
-                demisto.get(activity, 'AdditionalInformation.RemediationBlocklistUpdate')),
-            'AskTheAnalyst': {'Replies': demisto.get(activity, 'AdditionalInformation.AskTheAnalyst.Replies')},
-            'Mail': extract_mail(demisto.get(activity, 'AdditionalInformation.Mail.Replies')),
-            'ReadBy': demisto.get(activity, 'ReadBy')
+    if not response:
+        demisto.results('Alert activities were not found.')
```

Comment:
I am afraid this could break BC. What do you think?

Reply:
if there is no response, there is nothing to iterate on in response, which can reproduce an error as well.